core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -712,6 +746,184 @@ export class LoggedInUser extends BaseUser {
+   */
+  async subscribeToCreator(username: string): Promise<void> {
+    const profilePageUrl = `${profilePageUrlPrefix}/${username}`;
+
+    if (this.page.url() !== profilePageUrl) {
+      await this.navigateToProfilePage(username);
+    }
+
+    await this.clickOn(subscribeButton);
+    await this.page.waitForSelector(unsubscribeLabel);
+    showMessage(`Subscribed to the creator with username ${username}`);
+  }
+
+  /**
+   * This function checks if the number of subscribers equal to.
+   */
+  async expectNumberOfSubscribersToBe(number: number): Promise<void> {
+    const subscriberCount = await this.page.$eval(
+      '.e2e-test-oppia-total-subscribers',
+      element => element.textContent
+    );
+
+    if (subscriberCount && parseInt(subscriberCount) === number) {
+      showMessage(`Number of subscribers is equal to ${number}`);
+    } else {
+      throw new Error(`Number of subscribers is not equal to ${number}`);
+    }
+  }
+
+  /**
+   * Function for refreshing the page.
+   */
+  async refreshPage(): Promise<void> {
+    await this.page.reload();
+  }
+
+  /**
+   * Function for opening the subscribers tab.
+   */
+  async openSubscribersTab(): Promise<void> {
+    if (this.page.url() !== creatorDashboardUrl) {
+      await this.navigateToCreatorDashboardPage();
+    }
+
+    await this.page.click(subscriberTabButton);
+    await this.page.waitForSelector('.e2e-test-subscription-card');
+  }
+
+  /**
+   * This function checks whether given user is a subscriber or not.
+   */
+  async expectUserToBeASubscriber(username: string): Promise<void> {
+    let truncatedUsername = username;
+    if (username.length > 10) {
+      const ellipsis = '...';
+      truncatedUsername =
+        username.substring(0, 10 - ellipsis.length) + ellipsis;
+    }
+
+    const subscribers = await this.page.$$(subscriberCard);
+
+    if (subscribers.length === 0) {
+      throw new Error(`User "${username}" is not subscribed.`);
+    }
+
+    const subscriberUsername = await subscribers[0].$eval(
+      '.e2e-test-subscription-name',
+      element => (element as HTMLElement).textContent?.trim()
+    );
+
+    if (truncatedUsername === subscriberUsername) {
+      showMessage(`User ${username} is a subscriber`);
+    } else {
+      throw new Error(`User ${username} is not a subscriber`);
+    }
+  }
+
+  /**
+   * Checks whether the exploration with the given title is authored by the creator.
+   */
+  async expectExplorationWithTitleToBePresentInProfilePage(
+    title: string
+  ): Promise<void> {
+    const explorations = await this.page.$$(explorationCard);
+
+    if (explorations.length === 0) {
+      throw new Error('There are no explorations authored by the creator');
+    }
+
+    const explorationTitle = await explorations[0].$eval(
+      '.e2e-test-exp-summary-tile-title span',
+      element => (element as HTMLElement).textContent
+    );
+
+    if (explorationTitle === explorationTitle) {
```

Comment:
This will always evaluate to `true`. Are you trying to assert that the `explorationTitle` from the UI matches the given `title` parameter?

Reply:
yes they should match

Reply:
Oh, my bad. yes it should be title